### ISSUE TITLE
add import statement to import specified command in the session

### ISSUE THREAD

**Issue by p0W3RH311** [PowerShell/PowerShell#11762]
### steps to reproduce

to import from testmodule module only **get-testmodule** and **set-testmodule** in lexical scope

```
using module testmodule import get-testmodule,set-testmodule 

$a = get-testmodule 
```

**export-modulemember** is great but suppose if module not contains **export-modulemember**

**Comment by iSazonov** [comment]
You could use `Import-Module -Cmdlet get-testmodule,set-testmodule `

**Comment by mklement0** [comment]
Indeed, @iSazonov, but I can see how having the same ability in the context of the - generally preferable - `using module` statement would be helpful, if only to signal explicitly what commands are being used.

However, to fully mirror `Import-Module`, we'd either have to mirror the individual `-Cmdlet`, `-Function`, `-Alias` and `-Variable` parameters - which could get syntactically awkward - or assume that exported member names are unique across member _types_ (which is a sensible assumption).

Either way, note that neither a selectively importing `Import-Module` nor a `using module` would necessarily _prevent availability_ of those members that _aren't_ explicitly imported:

* If someone else has previously imported the same module (non-selectively, in the same scope domain), all exported members would already be available (though `Import-Module -Force -Cmdlet...` could override that; `using module` has no force-reloading option).

* For auto-loading modules, referencing not explicitly imported members still imports them on demand.

**Comment by iSazonov** [comment]
>  if only to signal explicitly what commands are being used

It seems this could be only achieved:
- by module manifest
- by JEA

**Comment by mklement0** [comment]
@iSazonov, this issue is about _voluntarily_ selecting only a _subset_ of a given module's exported members _at import time_, so module manifests and JEA don't apply.

By "signal" I meant "signal to the human reader", because, as discussed in my previous comment, you cannot reliably ensure that the scope trying to do a selective `using module` won't see _all_ exported members nonetheless.

Note that there's no _lexical_ scoping with modules (as there generally isn't in PowerShell).
`using module` invariably imports scope-domain-wide, only with `Import-Module -Scope` do you get _some_ control, but the scoping is always _dynamic_.

In short:

* `Import-Module` already supports selective importing (`-Cmdlet`, `-Function`, ...); there's a case to be made that `using module` should support the same.

* In both cases, non-import of the non-selected members cannot be _guaranteed_: it only works if no other code in the same scope domain imports the same module (non-selectively), and never with auto-loading modules. As such, at least without lexical scoping support, selective importing is primarily a (self-documenting) signal of _intent_ rather than something _enforced by the engine_.
